YIF.Core.Domain/Repositories/InstitutionOfEducationModeratorRepository.cs

```diff
@@ -53,9 +53,10 @@ public async Task<IEnumerable<InstitutionOfEducationModeratorDTO>> GetAllUniMode
             return _mapper.Map<IEnumerable<InstitutionOfEducationModeratorDTO>>(moderators.AsEnumerable());
         }
 
-        public Task<InstitutionOfEducationModeratorDTO> GetById(string id)
+        public async Task<InstitutionOfEducationModeratorDTO> GetById(string id)
```

Comment:
Change method name to  GetByUserId(string userId)

Reply:
Changed